druid/src/widget/button.rs

```diff
@@ -52,6 +52,27 @@ impl<T: Data> Button<T> {
         }
     }
 
+    /// Create a new button with the provided label.
+    ///
+    /// Use the `.on_click` method to provide a closure to be called when the
```

Comment:
and I'd probably link to `on_click` here? I'm trying to err on the side of linking to things from docs, I think it's ultimately a better experience.

Reply:
Sure, I'll update the doc comment on `Button::new` as well